test/tpu/xla_test_job.yaml

```diff
@@ -29,6 +29,9 @@ spec:
     - |
       python3 pytorch/xla/test/pjrt/test_operations.py -v
       python3 pytorch/xla/test/pjrt/test_experimental_pjrt_tpu.py
+      python3 pytorch/xla/test/spmd/test_xla_sharding.py
+      python3 pytorch/xla/test/spmd/test_xla_virtual_device.py
+      python3 pytorch/xla/test/spmd/test_train_spmd_linear_model.py
```

Comment:
Just a note: this will run as a single-device program without the `--sharding` flag. You can add batch sharding with `--sharding batch` or model sharding with `--sharding megatron-lm`.

Reply:
Thanks for the tip! I wonder which tests are worth including for CI given that we want to limit the runtime: single device, batch sharding, and/or model sharding. cc @yeounoh

Reply:
Thanks @steventk-g , adding a comment post review/merge. I think we could run `test_train_spmd_linear_model` in XL ML Test infra instead. A simple training unit tests is included in `test_xla_sharding` for CI.